README.md

```diff
@@ -52,6 +52,16 @@ In case you use an exception reporting service (e.g. Bugsnag) you might want to
 define an error handler. See [Customizing the error
 handler](#customizing-the-error-handler) for more information.
 
+**ActiveJob Dependency**
```

Comment:
I think elsewhere in this document we've split the camel case words, I think we should do the same here.

Also my markdown linter tells me this should be a header instead of emphasis.

```suggestion
#### Active Job Dependency
```